src/mmw/sass/base/_header.scss

```diff
@@ -132,6 +132,14 @@ header {
         width: 100px;
     }
 
+    #download-cc-gms {
```

Comment:
This renders on top of "Current Conditions" on Mac chrome + FF

Instead of trying to force this element out of the `.toolbar-region` bounds, we should have the `.toolbar-region` abut the `.scenarios-region`, and relatively position "export" and "add changes to this area" inside it.
![screen shot 2017-08-31 at 10 48 00 am](https://user-images.githubusercontent.com/7633670/29930975-eb88eafa-8e3d-11e7-8e7a-761c888c2fc8.png)

Reply:
There is some unusual CSS in the headers that made this difficult - if you've got an idea that works, I'd be happy for you to demonstrate it to me.

Reply:
I think something like this will work:
```diff
diff --git a/src/mmw/js/src/modeling/templates/scenarioAddChangesButton.html b/src/mmw/js/src/modeling/templates/scenarioAddChangesButton.html
index faf2e23..02f2601 100644
--- a/src/mmw/js/src/modeling/templates/scenarioAddChangesButton.html
+++ b/src/mmw/js/src/modeling/templates/scenarioAddChangesButton.html
@@ -1,14 +1,16 @@
-<div class="tab-content scenario-toolbar-tab-content"></div>
-{% if isOnlyCurrentConditions %}
-    {% if isGwlfe %}
-        <button id="download-cc-gms" class="btn btn-link" type="button" aria-expanded="true">
-            <i class="fa fa-download"></i> Export GMS
-        </button>
+{% if isOnlyCurrentConditions and isGwlfe %}
+<button id="download-cc-gms" class="btn btn-link" type="button" aria-expanded="true">
+    <i class="fa fa-download"></i> Export GMS
+</button>
 
-        {% include "./gmsDownloadForm.html" %}
-    {% endif %}
+{% include "./gmsDownloadForm.html" %}
 
+{% endif %}
+<div class="right-controls-container">
+    <div class="tab-content scenario-toolbar-tab-content"></div>
+    {% if isOnlyCurrentConditions %}
     <button id="add-changes" class="btn btn-link" type="button" aria-expanded="true">
         <i class="fa fa-plus-circle"></i> Add changes to this area
     </button>
-{% endif %}
+    {% endif %}
+</div>
diff --git a/src/mmw/js/src/modeling/views.js b/src/mmw/js/src/modeling/views.js
index fac5d51..db70a6a 100644
--- a/src/mmw/js/src/modeling/views.js
+++ b/src/mmw/js/src/modeling/views.js
@@ -790,6 +790,7 @@ var ScenarioToolbarView = Marionette.CompositeView.extend({
     template: scenarioAddChangesButtonTmpl,
     collection: models.ScenariosCollection,
     childViewContainer: '.tab-content.scenario-toolbar-tab-content',
+    className: 'toolbar-container',
 
     ui: {
         addChangesButton: '#add-changes',
diff --git a/src/mmw/sass/base/_header.scss b/src/mmw/sass/base/_header.scss
index a96a55a..8081a32 100644
--- a/src/mmw/sass/base/_header.scss
+++ b/src/mmw/sass/base/_header.scss
@@ -132,14 +132,6 @@ header {
         width: 100px;
     }
 
-    #download-cc-gms {
-      position: absolute;
-      left: 170px;
-      top: 8px;
-      font-size: 13px;
-      color: inherit;
-    }
-
     #gwlfe-modifications-bar, #gwlfe-modifications-popup {
       padding: 5px;
       position: absolute;
@@ -213,13 +205,30 @@ header {
         white-space: nowrap;
     }
 
+
     #toolbar-region {
-        margin-right: 0;
-        margin-left: auto;
+        width: 100%;
+    }
+
+    #toolbar-region > .toolbar-container {
+        display: flex;
+        flex-basis: 100%;
+        flex-direction: row;
+        height: 100%;
+
+        .right-controls-container {
+            margin-left: auto;
+        }
+
         #add-changes {
-          position: relative;
-          top: -2px;
-          color: $brand-primary;
+            color: $brand-primary;
+            height: 100%;
+        }
+
+        #download-cc-gms {
+            font-size: 13px;
+            height: 100%;
+            color: inherit;
         }
     }

```